### DISCUSSION TITLE
If it's possible to embed a content of code file as a code block inside a markdown blog/doc?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [facebook/docusaurus#9382] [Q&A] [answered]
I have a blog with this file structure

```
.
├── index.md
└── te-quick-start
    └── index.html
```

I want to put the content of the index.html as a code block inside index.md

like:

```md index.md
[code block](index.html)
```

Then it will render out like I literally put the content of the index.html as a code block inside the `index.md` file

**Comment by slorber** [comment, marked as answer]
Documented here: https://docusaurus.io/docs/markdown-features/react#importing-code-snippets
